appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -146,27 +147,50 @@
 		return createOkResponse(userService.getPublicUser(userId), false);
 	}
 
-	@RequestMapping(value = "/{userId}/apple.mobileconfig", method = RequestMethod.GET)
+	@RequestMapping(value = "/{userId}/apple.mobileconfig", params = { "sign" }, method = RequestMethod.GET)
 	@ResponseBody
 	public ResponseEntity<byte[]> getAppleMobileConfig(
-			@RequestHeader(value = Constants.PASSWORD_HEADER) Optional<String> password, @PathVariable UUID userId)
+			@RequestHeader(value = Constants.PASSWORD_HEADER) Optional<String> password,
+			@RequestParam(value = "sign", defaultValue = "false") String signStr, @PathVariable UUID userId)
 	{
 		HttpHeaders headers = new HttpHeaders();
 		headers.setContentType(new MediaType("application", "x-apple-aspen-config"));
+		boolean mustSign = Boolean.TRUE.toString().equals(signStr);
 		try (CryptoSession cryptoSession = CryptoSession.start(password, () -> userService.canAccessPrivateData(userId)))
 		{
-			return new ResponseEntity<>(getUserSpecificAppleMobileConfig(userService.getPrivateUser(userId)), headers,
+			return new ResponseEntity<>(getUserSpecificAppleMobileConfig(userService.getPrivateUser(userId), mustSign), headers,
 					HttpStatus.OK);
 		}
 	}
 
-	private byte[] getUserSpecificAppleMobileConfig(UserDto privateUser)
+	@RequestMapping(value = "/{userId}/apple.mobileconfig", method = RequestMethod.GET)
```

Comment:
I added this because otherwise it would add the ``?sign=...`` parameter to the ``yona:appleMobileConfig`` link.

The ``sign`` query parameter will be taken out when we close this feature, so this is just temporarily.